fetch/stale-while-revalidate/fetch.html

```diff
@@ -17,9 +17,12 @@
   var request_token = token();
 
   const response = await fetch(`resources/stale-script.py?token=` + request_token);
+  const body = await response.text();
   const response2 = await fetch(`resources/stale-script.py?token=` + request_token);
+  const body2 = await response2.text();
```

Comment:
Let's move this line between the two asserts.

Reply:
Sure, done.